app/views/validation.scala.html

```diff
@@ -37,6 +37,12 @@
             </div>
             <div id="upper-menu-holder">
                 <div id="upper-menu-title-bar"></div>
+                <div id="label-control-holder"></div>
+                <div id="label-control-holder">
+                    <button id="label-control-button" class="button label-control-button" title="Press the &quot;H&quot; key" data-toggle="tooltip" data-placement="top">
+                        <img src='@routes.Assets.at("javascripts/SVLabel/img/icons/ZoomIn.svg")' class="label-button-icon" alt="Hide Label">
```

Comment:
Needs something other than the zoom in image

Reply:
@misaugstad Sent an image in the GitHub task thread that seems to be good. Will incorporate it into the interface once you give me the go ahead.